checker/src/org/checkerframework/checker/lock/LockAnnotatedTypeFactory.java

```diff
@@ -41,22 +57,23 @@
     extends GenericAnnotatedTypeFactory<CFValue, LockStore, LockTransfer, LockAnalysis> {
 
     /** Annotation constants */
-    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE;
+    protected final AnnotationMirror LOCKHELD, LOCKPOSSIBLYHELD, SIDEEFFECTFREE, GUARDEDBYINACCESSIBLE, GUARDEDBY, GUARDEDBYBOTTOM, JCIPGUARDEDBY, JAVAXGUARDEDBY, GUARDSATISFIED;
```

Comment:
I can try making them aliases of our GuardedBy annotation and see whether javax and jcip GuardedBy still work as @Holding annotations on methods as well. If they don't, I think we should drop support for them.